packages/blitz-rpc/src/data-client/rpc.ts

```diff
@@ -93,18 +95,29 @@ export function __internal_buildRpcClient({
       serialized = serialize(params)
     }
 
+    const body = JSON.stringify({
+      params: serialized.json,
+      meta: {
+        params: serialized.meta,
+      },
+    })
+
+    if (httpMethod === "GET") {
+      fullRoutePath +=
+        "?" +
+        new URLSearchParams({
+          params: stringify(serialized.json),
+          meta: stringify(serialized.meta),
+        })
```

Comment:
Here's your `meta` is an array bug. 

each time this function runs, it keeps appending to fullRoutePath, which is "global" at the module level.